.github/workflows/ci.yaml

```diff
@@ -276,4 +276,40 @@ jobs:
       - name: kube-webhook-certgen image build
         if: ${{ steps.filter-images.outputs.kube-webhook-certgen == 'true' }}
         run: |
-          cd images/kube-webhook-certgen && make build
\ No newline at end of file
+          cd images/kube-webhook-certgen && make build
+
+  test-image:
```

Comment:
nit here: test-image-what? Would be good to explain we are testing image-webhook.

Approving and lgtming here anyway, I will put a hold if you think this deserves to be fixed, otherwise just remove the hold !

Thanks

Reply:
Hm, the section above is named `test-image-build`, maybe this should be named `image-test` then? To indicate that we run tests for images?

I assume `test-image-build` was previously used only for building test image and then evolved into building other images as well, without being renamed?